.github/workflows/run-tests-linux-multiarch.yml

```diff
@@ -13,7 +14,6 @@ jobs:
   # Build testing image that would be used to build and run against different platforms
   # Currently only Linux x64 is tested
   build-image:
-    if: "github.event_name == 'pull_request' || ((github.event_name == 'schedule' || github.event_name == 'workflow_call') && github.repository == 'scala-native/scala-native')"
```

Comment:
Removing this would trigger scheduled builds in all forks of the project, we need to keep it verbose unfortunetlly so that the nightly builds which enable some additional tests would not be started outside the main repo 
`github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' || (...)`

```

Reply:
In fact they're still triggered but are stopped quickly: https://github.com/WojciechMazur/scala-native/actions?query=event%3Aschedule

Reply:
@WojciechMazur do you mind to reduce this if to:
```
    if: "!(github.event_name == 'schedule' && github.repository != 'scala-native/scala-native')"
```
that should make it easy to support and enable schedule build only for main repository.

but current way probably cleaner and much easy to understand and support.

Reply:
sure, looks good

Reply:
@WojciechMazur after some thought I decided to move forward with old way. Suggested way isn't clear and may lead to future errors when someone tries to change it.